db/schema.rb

```diff
@@ -128,6 +128,15 @@
     t.index ["poa_code"], name: "index_accredited_organizations_on_poa_code", unique: true
   end
 
+  create_table "accredited_representative_portal_verified_representatives", force: :cascade do |t|
```

Comment:
Please merge from master

Reply:
I will approve after merging from master

Reply:
Thanks, @RachalCassity; just [merged main](https://github.com/department-of-veterans-affairs/vets-api/pull/16493/commits/b3def1cfe5a926e804c877420e572187e4fbf52d), which removed the migrations from this PR. Ready for review!